libraries/AP_GPS/AP_GPS.h

```diff
@@ -305,8 +305,13 @@ class AP_GPS
     }
 
     // the expected lag (in seconds) in the position and velocity readings from the gps
-    float get_lag(uint8_t instance) const;
-    float get_lag(void) const { return get_lag(primary_instance); }
+    // return true if the GPS hardware configuration is known or the lag parameter has been set manually
+    bool get_lag(uint8_t instance, float &lag_sec) const;
+    bool get_lag(float &lag_sec) const
+    {
+        bool is_valid = get_lag(primary_instance, lag_sec);
```

Comment:
Why not just ```{ return get_lag(primary_instance, lag_sec) }``` it would be more inline with the way the rest of the file is structured.